src/components/limited-restoration-panel/LimitedRestorationPanel.vue

```diff
@@ -1,127 +1,148 @@
 <template id="limited-restoration-panel">
-  <v-radio-group class="mt-0 pt-0 ml-8" v-model="selectMonths" mandatory>
-    <v-radio class="radio-button pt-2" id="twentyFour-radio" name="selectMonths" label="2 years" :value=24 />
-    <v-radio class="radio-button pt-2" id="eighteen-radio" name="selectMonths" label="18 months" :value=18 />
-    <v-radio class="radio-button pt-2" id="twelve-radio" name="selectMonths" label="12 months" :value=12 />
-    <v-radio class="radio-button pt-2" id="six-radio" name="selectMonths" label="6 months" :value=6 />
+  <v-radio-group
+    v-model="radioValue"
+    class="mt-0 pt-0 ml-8"
+    mandatory
+  >
+    <v-radio
+      id="radio-24"
+      class="radio-button pt-2"
+      name="radioValue"
+      label="2 years"
+      value="24"
+    />
+    <v-radio
+      id="radio-18"
+      class="radio-button pt-2"
+      name="radioValue"
+      label="18 months"
+      value="18"
+    />
+    <v-radio
+      id="radio-12"
+      class="radio-button pt-2"
+      name="radioValue"
+      label="12 months"
+      value="12"
+    />
+    <v-radio
+      id="radio-6"
+      class="radio-button pt-2"
+      name="radioValue"
+      label="6 months"
+      value="6"
+    />
     <v-row class="ml-0 mt-0 radio-button pt-2">
-      <v-radio class="mt-n4" id="custom-months" name="selectMonths" value="customMonths" />
+      <v-radio
+        id="radio-custom"
+        class="mt-n4"
+        name="radioValue"
+        value="customMonths"
+      />
       <v-form ref="monthsRef">
         <v-text-field
-          id="months-text-field"
-          class="number-months-field"
+          id="text-field-months"
+          v-model="inputValue"
+          class="text-field-months"
           type="number"
           dense
+          filled
           hide-spin-buttons
-          :rules="monthRules"
-          v-model="numberOfMonths"
-          :disabled="!customMonths"
-          @change="onMonthsChanged"
-          filled />
+          :rules="monthsRules"
+          :disabled="(radioValue !== 'customMonths')"
+          @input="onMonthsInput($event)"
```

Comment:
What's the difference between `@change` and `@input`?

Reply:
`@change` fires when the value has changed (been accepted by the component).

`@input` fires when anything in the input has changed, such as typing in "1" or backspacing over it. This allows me to pass this component's "real-time" validity to the parent, since there's no Done button.

Reply:
I wish I knew this earlier 😭 I've struggled with real-time validity on various occasions. Alright, noted. Thanks Sev!

Reply:
For the record, native HTML controls have a ton of events that you can research (Google).

This is a Vuetify component and I didn't find a ton of good info on events, but I played with the documented ones and figured out what they did. (Ref: https://wotamann.gitbook.io/vuetify-form-base/schema/events)

Reply:
I just found this but it doesn't change my code: https://vuetifyjs.com/en/api/v-text-field/#props-validation-value

Here's all the docs have to say about text field events: https://vuetifyjs.com/en/api/v-text-field/#events

I guess you're supposed to guess that text fields inherit base events from the form 🤷‍♂️  😒

Reply:
One thing that I've noticed is that when it comes to Vuetify (or Vue in general), there isn't much information out there as React or Native HTML stuff for example. I would have to dig deep in the docs to find an answer to whatever that I'm trying to achieve. 

This here is a perfect example of that. Thanks for the Refs Sev. Also, I'll be keeping this in mind so that I won't have to guess next time 😟